host/containerinit/init.go

```diff
@@ -399,29 +401,39 @@ func setupCommon(c *Config, log log15.Logger) error {
 		return err
 	}
 
-	setupLimits(log)
+	setupLimits(c, log)
 
 	return nil
 }
 
-const RLIMIT_NPROC = 6
-
-func setupLimits(log log15.Logger) {
-	type rlimit struct {
-		resource int
-		value    uint64
-	}
+const (
+	RLIMIT_NPROC          = 6
+	defaultMaxFD    int64 = 10000
+	defaultMaxProcs int64 = 256
```

Comment:
Since we have defaults higher in the stack, we probably don't need to duplicate them here and can leave them to default if not specified.